### ISSUE TITLE
Header doesn't behave right?

### ISSUE THREAD

**Issue by ZeroSoulEater** [react-native-training/react-native-elements#677]
this is the component
```
<View style={{flex: 1}}>
       <Header outerContainerStyles={{flex:0}}
                backgroundColor='#d80030'
                leftComponent={this.renderBack()}
        />
        <KeyboardAvoidingView style={styles.end} behavior='padding'>
          <FlatList style={{backgroundColor:'#0000FF', flex:1}} ...>
          ......
```
Header is rendert like its not on the screen (you can see it as long flatlist doesn't have a background color) my Flatlist is always a layer over the Header and if i put a button in the Header to switch a view back i can't click it because my list in the foreground.

**Comment by iRoachie** [comment]
Hey @ZeroSoulEater a fix to the header was made in https://github.com/react-native-training/react-native-elements/commit/588c162d4bca74830aa9fe17eea1b20852e3f0d1. However that new version isn't posted as yet. You can use the `next` branch in your package.json until we release that version

**Comment by iRoachie** [comment]
Fixed in v18.0.0
